File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int ans=0;
        for(int i=1;i<nums.size();i++){
            if(nums[i]==nums[i-1]){
                nums[i-1]=101;
                ans++;
            }
        }
        sort(nums.begin(),nums.end());
        int i = 0;
        for(i=0;i<nums.size();i++){
            if(nums[i]==101) break;
        }
        return i;
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int i=0,j=0;
        if(nums.length==1) return 1;
        while(j+1<nums.length){
            if(nums[j]!=nums[j+1]){
                nums[i]=nums[j];
                i++;
            }
            j++;
        }
        nums[i]=nums[j];
        return i+1;
    }
}